app.js

```diff
@@ -5,6 +5,7 @@ var qs = require("querystring");
 var requestLib = require("request");
 var Rooms = require("./lib/rooms");
 var HookHandler = require("./lib/hook-handler");
```

Comment:
As you said, given this is handling Slack requests only, the class name should be indicative of this.